src/main/kotlin/ch/wisv/choice/security/CHConnectSecurityConfiguration.kt

```diff
@@ -70,10 +71,12 @@ class CHConnectSecurityConfiguration(private val properties: CHConnectConfigurat
                 .and()
                     .authorizeRequests()
                     .antMatchers("/dashboard/**").hasRole("ADMIN")
+                    .antMatchers("/api/**").hasRole("USER")
                     .anyRequest().permitAll()
                 .and()
                     .logout()
                     .logoutSuccessUrl("/")
+                .and().csrf().disable()
```

Comment:
You will want to disable CSRF protection on just the API, not the admin dashboard.